packages/ckeditor5-dev-env/tests/translations/collect-utils.js

```diff
@@ -16,16 +16,42 @@ const glob = require( 'glob' );
 const fs = require( 'fs-extra' );
 
 describe( 'collect-utils', () => {
-	let sandbox;
+	const sandbox = sinon.sandbox.create();
```

Comment:
At least we can remove `sandbox.restore()` if we're assigning new sandbox to the variable before each test.